doc/src/documentation_style_guide.rst

```diff
@@ -0,0 +1,1019 @@
+* Development policies (for example: https://github.com/sympy/sympy/wiki/Python-version-support-policy)
+* Release notes (for example: https://github.com/sympy/sympy/wiki/Release-Notes-for-1.5)
+* Various pages that different contributors have added
+
+Narrative Documentation Guidelines
+==================================
+
+Extensive documentation, or documentation that is not centered around an API
+reference, should be written as a narrative document in the Sphinx docs (located
+in the `doc/src directory
+<https://github.com/sympy/sympy/tree/master/doc/src>`_). The narrative documents
+do not reside in the Python source files, but as standalone restructured files
+in the doc/src directory. SymPy’s narrative documentation is defined as the
+collective documents, tutorials, and guides that teach users how to use SymPy.
+
+Steps to Writing Documentation
+------------------------------
+
+The first step to contributing to the code base is creating your development
+environment. Please find instructions on how to create your development
+environment in our `Development Workflow – Create Your Environment
+<https://github.com/sympy/sympy/wiki/Development-workflow#create-your-environment>`_
+guide.
+
+Once you have created your development environment, follow these steps:
+
+1. Installation
+---------------
+
+**Debian/Ubuntu**
+
+For Debian/Ubuntu, install the prerequisites::
+
+   apt-get install python-sphinx texlive-latex-recommended dvipng librsvg2-bin
+   imagemagick docbook2x graphviz python -m pip install sphinx-math-dollar
+
+And do::
+
+   make html
+
+If you get mpmath error, install python-mpmath package::
+
+   apt-get install python-mpmath
+
+If you get matplotlib error, install python-matplotlib package::
+
+   apt-get install python-matplotlib
+
+And to view it, do::
+
+   firefox _build/html/index.html
+
+**Fedora**
+
+For Fedora (and maybe other RPM-based distributions), install the
+prerequisites::
+
+   dnf install python3-sphinx librsvg2 ImageMagick docbook2X texlive-dvipng-bin
+   texlive-scheme-medium librsvg2-tools python -m pip install sphinx-math-dollar
+
+After that, run::
+
+   make html
+
+If you get mpmath error, install python3-mpmath package::
+
+   dnf install python3-mpmath
+
+If you get matplotlib error, install python3-matplotlib package::
+
+   dnf install python3-matplotlib
+
+And view it at::
+
+  _build/html/index.html
+
+**Mac**
+
+For Mac, first install homebrew: https://brew.sh/
+
+Then install these packages with homebrew::
+
+  brew install imagemagick graphviz docbook librsvg
+
+Install these packages with either pip or conda::
+
+  python -m pip install mpmath matplotlib sphinx sphinx-math-dollar
+
+Or::
+
+  conda install -c conda-forge mpmpath matplotlib sphinx sphinx-math-dollar
+
+**Windows 10**
+
+Making your Sphinx build successful on the Windows system is tricky because
+some dependencies like ``dvipng`` or ``docbook2x`` are not available.
```

Comment:
What exactly are these needed for? Can we maybe eliminate that dependency?

Reply:
Maybe to build some of the things that require graphviz.

Reply:
Maybe @smichr knows if this recommendation is accurate. I think he uses Windows. At any rate, I believe this text is from the doc/README, so I would just leave it as is unless we know for sure it is incorrect.